app/tests/pages/view-application.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
Can you move this test to the `tests/unit` folder?

Reply:
Moved to unit test folder.